src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/flow_units/ResourceFlowUnit.java

```diff
@@ -128,16 +130,18 @@ public static ResourceFlowUnit buildFlowUnitFromWrapper(final FlowUnitMessage me
   public List<Field<?>> getSqlSchema() {
     List<Field<?>> schema = new ArrayList<>();
     if (!this.isEmpty()) {
-      schema.add(DSL.field(DSL.name(SQL_SCHEMA_CONSTANTS.TIMESTAMP_COL_NAME), String.class));
+      schema.add(ResourceFlowUnitFieldValue.TIMESTAMP_FIELD.getField());
+      schema.add(ResourceFlowUnitFieldValue.RCA_NAME_FILELD.getField());
       schema.addAll(this.getResourceContext().getSqlSchema());
     }
     return schema;
   }
 
-  public List<Object> getSqlValue() {
+  public List<Object> getSqlValue(String rcaName) {
     List<Object> value = new ArrayList<>();
```

Comment:
Can the list contain values other than strings ?

Reply:
Yes, we can have Double and Integer as well.